whois-commons/src/main/java/net/ripe/db/whois/common/DataSourceConfigurations.java

```diff
@@ -55,6 +55,20 @@ public ComboPooledDataSource nrtmSlaveDataSource(@Value("${nrtm.slave.database.u
         return createDataSource(DRIVER_CLASS_NAME, jdbcUrl, jdbcUser, jdbcPass);
     }
 
+    @Bean
+    public ComboPooledDataSource nrtmClientMasterDataSource(@Value("${nrtm.client.database.url}") final String jdbcUrl,
```

Comment:
Whois-nrtm-client needs to be independent, remove it from here and put the NRTM client specific configuration in new module